_errata/print01-ch11-c_void.md

```diff
@@ -0,0 +1,20 @@
+---
+chapter: 11
+page: 207
+kind: inaccuracy
+date: 2022-02-01
+---
+The sentence "Since `Foo` and `Bar` are zero-sized types, they can be used in
+place of `()` in the `extern` method signatures." in the final paragraph of the
```

Comment:
```suggestion
The sentence

> Since `Foo` and `Bar` are zero-sized types, they can be used in place of `()` in the `extern` method signatures.

in the final paragraph of the
```

Reply:
(modulo text wrapping)